.github/workflows/release.yml

```diff
@@ -27,8 +27,26 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
           aws-region: us-east-1
+  
+  cli-token-validation:
+    needs: [ cli-core-token-validation ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout cli-core repo
+        uses: actions/checkout@v2
+      - name: Execute py script to validate twilio-cli tokens
+        id: cli_token  
+        run: |
+          output=$(python3 .github/scripts/validate_cli_tokens.py)
+          echo "::set-output name=tokenStatus::$output"
+      - name: Print status
+        run: echo "${{ steps.cli_token.outputs.tokenStatus }}"
+      - name: Validate the github workflow
+        if: ${{ steps.cli_token.outputs.tokenStatus != 'success'}}
+        run: exit 1
```

Comment:
Force failure if the status is not complete

Reply:
@shamantraghav Can you please remove hardcoded twilio and use github.repository_owner from both the commits